src/main/java/com/learn/assignment/Package.java

```diff
@@ -0,0 +1,63 @@
+package com.learn.assignment;
+
+import java.util.Objects;
+
+class Package implements Comparable<Package>{
+    String id;
+    double weight;
+    String city;
+
+    public Package(String id, double weight, String city) {
+        //System.out.println("------------In Package()--------------");
```

Comment:
Why does this code need commented line ?

Reply:
//System.out.println("------------In Package()--------------");
Removed the commented code.